tests/ui/macros/rfc-3086-metavar-expr/issue-111905.stderr

```diff
@@ -0,0 +1,14 @@
+error: related fragment that refers the `count` meta-variable expression was not found
+  --> $DIR/issue-111905.rs:4:26
+   |
+LL |     ($($t:ident)*) => { ${count(t, 4294967296)} };
+   |                          ^^^^^^^^^^^^^^^^^^^^^^
+
+error: related fragment that refers the `count` meta-variable expression was not found
+  --> $DIR/issue-111905.rs:9:60
+   |
+LL |     ( $( { $( [ $( ( $( $t:ident )* ) )* ] )* } )* ) => { ${count(t, 4294967296)} }
+   |                                                            ^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Do we have enough information to point to the `$` in the pattern which corresponds to the current level?

Reply:
Well, I wasn't able to capture the pattern span after ~1 hour of attempts